script.js

```diff
@@ -2,54 +2,81 @@
 // prepare DOM Elements
     const todoInput = document.querySelector('.todo-input')
```

Comment:
Use a more efficient method to query DOM elements to enhance the code readability and performance.

```
// Before
const todoInput = document.querySelector('.todo-input');
const ulList = document.querySelector('.todolist ul');

// After
const todoInput = document.getElementsByClassName('todo-input')[0];
const ulList = document.getElementsByClassName('todolist')[0].getElementsByTagName('ul')[0];

```